cli/loader_lib.c

```diff
@@ -153,6 +152,181 @@ JL_DLLEXPORT const char * jl_get_libdir()
     return lib_dir;
 }
 
+#ifdef _OS_LINUX_
+#ifndef GLIBCXX_LEAST_VERSION_SYMBOL /* Should always be defined in the makefile. This appeases the linter. */
+#define GLIBCXX_LEAST_VERSION_SYMBOL "GLIBCXX_a.b.c"
+#endif
+
+#include <link.h>
+#include <sys/wait.h>
+
+static void write_wrapper(int fd, char *str, size_t len)
+{
+    size_t written_sofar = 0;
+    while (len) {
+        ssize_t bytes_written = write(fd, str + written_sofar, len);
+        if (bytes_written == -1 && errno == EINTR) continue;
+        if (bytes_written == -1 && errno != EINTR) {
+            perror("Error in write wrapper:\nwrite");
```

Comment:
"write wrapper" is a meaningless phrase to users. While this is unlikely to fail (probably meaning the parent was killed), it is clearer if it specifies that it comes from julia and is part of the libstdcxxprobe helper functionality
```suggestion
            perror("(julia) child libstdcxxprobe write:");
```